diff -Nurp gcc-7.2.0-orig/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc gcc-7.2.0/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc
--- gcc-7.2.0-orig/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc	2017-02-16 22:23:51.526073000 +0100
+++ gcc-7.2.0/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc	2020-03-15 19:23:10.459962487 +0100
@@ -154,7 +154,6 @@ typedef struct user_fpregs elf_fpregset_
 # include <sys/procfs.h>
 #endif
 #include <sys/user.h>
-#include <sys/ustat.h>
 #include <linux/cyclades.h>
 #include <linux/if_eql.h>
 #include <linux/if_plip.h>
@@ -247,7 +246,6 @@ namespace __sanitizer {
 #endif // SANITIZER_LINUX || SANITIZER_FREEBSD
 
 #if SANITIZER_LINUX && !SANITIZER_ANDROID
-  unsigned struct_ustat_sz = sizeof(struct ustat);
   unsigned struct_rlimit64_sz = sizeof(struct rlimit64);
   unsigned struct_statvfs64_sz = sizeof(struct statvfs64);
 #endif // SANITIZER_LINUX && !SANITIZER_ANDROID
